### PR TITLE
🚧  admin tab in mobile bottom navbar

### PR DESCRIPTION
**Pull request by PierreBrethes**
Fix issue : #347

### CODE REVIEW

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -1,101 +1,134 @@
...
+                  : `/icons/${darkMode ? "dark" : "light"}/zenika.svg`
               }
-              alt={"Line"}
+              alt={"admin"}
               width="25"
               height="25"
               className="p-1"
             />
-            <span className="text-center">{t("nav.search")}</span>
+            {/* {skills && (
```
Comment (PierreBrethes) [inline src/components/Navbar.tsx:113]: Seems a bit heavy just to have the number of skills an admin have to add, do i still import skills to get the little "notification" ? 
If yes it's easy and fast to add :)

Comment (benjaminbrgs) [reply src/components/Navbar.tsx:113]: Agree with you, a bit heavy for a little notification 😬 But it's requested, do you have another way in mind to do that ?

**Review by benjaminbrgs — APPROVED** [review]

File: src/components/Navbar.tsx (hunk 1 above)
Comment (PierreBrethes) [reply src/components/Navbar.tsx:113]: The best would be a new request, like a select count, so you would just receive a number

Comment (benjaminbrgs) [reply src/components/Navbar.tsx:113]: Sounds good, we can do this way

**Pull request merged**